stanford-earth/stanford_news
# EARTH-519 Earth news sub pages

# READY FOR REVIEW

# Summary
- Apply the news views library to any view that has the class instead of key off of the display name.
- set display flex since some items weren't flowing correctly. If you can think of a better way i'd be open to it. I'll attach an image to the conversation below

# Needed By (Date)
- end of sprint

# Urgency
- low

# Steps to Test

1. Checkout branch
2. clear cache
3. view page /news to make sure it still looks right.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
